### ISSUE TITLE
Bug related to `predict_proba`

### ISSUE THREAD

**Issue by jrzaurin** [jrzaurin/pytorch-widedeep#7]
Initially, the activation function for a multi-class problems was `F.softmax()`. This is not necessary when using `F.cross_entropy` since the latter already applies a `logSoftmax` to the inputs. 

I fix this for the `.fit` method but stills needs to be fixed for the `predict_proba` method. 

Note that this will not change the accuracy metric since `Softmax` does not change `argmax` (i.e preserves the rank order of the inputs).
